web/Migrations/20220825144120_AddHasGuarantorAndChangeEETInTenants.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class AddHasGuarantorAndChangeEETInTenants : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
```

Comment:
You can add something into the migration here to make the db invert all existing true and false values :)

Reply:
Moving my comment from below: 
I think technically to preserve the 'trueness' of our data, we'll want to negate any existing data in the column by adding a line to execute some SQL, like:
```
Sql("UPDATE Tenant SET NotInEET = 1- NotInEET;");
```
@oskarwilliams378 does this look right to you?

Reply:
(and we'd want to do the same in the `Down` method)